addons/mail/static/src/core/web/chat_window_service_patch.js

```diff
@@ -6,7 +6,11 @@ import { patch } from "@web/core/utils/patch";
 
 patch(ChatWindowService.prototype, {
     async _onClose(chatWindow, options) {
-        if (this.ui.isSmall && !this.store.discuss.isActive) {
+        if (
+            this.ui.isSmall &&
+            !this.store.discuss.isActive &&
+            (!chatWindow.thread || chatWindow.thread?.type !== "livechat")
```

Comment:
1. "livechat" shouldn't be referenced in mail
2. this is probably breaking standard flow that made use of this (it should re-open the menu when closing livechat chat window)

I'm thinking out loud but the proper fix might be to add an extra field on chat window when opening it, to determine whether the menu should be re-opened when closing it.